spec/migrations/20160414094300_change_capacity_to_hash_from_persistent_volume_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_migration
+
+describe ChangeCapacityToHashFromPersistentVolume do
+  let(:persistent_volume_stub) { migration_stub(:PersistentVolume) }
+
+  migration_context :up do
+    it "changes capacity to hash" do
```

Comment:
@alongoldboim do you think it might be good to test some edge cases like `nil` or hash with multiple values? These migrations are a one shot only 🔫 if something goes wrong, it will for all customers